esp/templates/users/profiles/usercontact.html

```diff
@@ -1,6 +1,6 @@
 <!-- User ContactInfo -->
 
-<h3>Contact Information</h3>
+<h3>{% ifequal profiletype "student" %}Student {% endifequal %}Contact Information</h3>
```

Comment:
What about this instead?
```suggestion
<h3>{{ profiletype|capfirst }} Contact Information</h3>
```